### ISSUE TITLE
Fix: handle google services files in expo env

### ISSUE THREAD

**Issue by zaniluca** [zaniluca/ping-4-gitlab#7]
## Summary
TLDR we need to keep google services files private; how do we achieve this? by adding them to `.gitignore`! The problem is that by doing this EAS doesn't pick them up when uploading to it.

The solution we have rn is to override the `.gitignore` with a `.easignore` which has everything ignored by the `.gitignore` except for the google services files.

This is not considered the best way to handle this situation as mentioned in the related issue; we should follow the solution in the related issue and handle everything with Environment variables in the EAS dashboard, this will benefit in both not having to keep a local copy of the services files and also not having to copy every change to the `.gitignore` to the `.easignore`.

When previously trying to use this solution it wasn't working as intended and the build services couldn't decode and copy the files from env.

## Todo / ordered steps
- [ ] Download and encode in Base64 the services files (`$ base64 google-services.json`)
- [ ] Copy the output and paste it into a secret in the EAS console named GOOGLE_SERVICES_ANDROID_BASE64
- [ ] Repeat the steps above for the ios one and change the name to GOOGLE_SERVICES_IOS_BASE64
- [ ] Delete `.easignore`
- [ ] Add a pre-install hook like this
```
"eas-build-pre-install": "echo $GOOGLE_SERVICES_ANDROID_BASE64 | base64 --decode > ./google-services.json && echo $GOOGLE_SERVICES_IOS_BASE64 | base64 --decode > ./GoogleService-Info.plist"
```
Everything should build without failing in the last steps; if it fails will probably be with a message "Can not find google-services.json / GoogleServices_Info.plist"

## Possible impacts / side effects on existing features
This will only impact the build phase and should not have any side effect

## Additional notes
EAS related issue: https://github.com/expo/eas-cli/issues/228
